easybuild/easyblocks/t/tensorflow.py

```diff
@@ -595,11 +595,12 @@ def configure_step(self):
             print_warning('Jemalloc is not supported in TensorFlow %s, the EC option with_jemalloc has no effect',
                           self.version)
         # Disable support of some features via config switch introduced in 1.12.1
-        if LooseVersion(self.version) >= LooseVersion('1.12.1'):
-            self.target_opts += ['--config=noaws', '--config=nogcp', '--config=nohdfs']
-            # Removed in 2.1
-            if LooseVersion(self.version) < LooseVersion('2.1'):
-                self.target_opts.append('--config=nokafka')
+        if LooseVersion(self.version) >= LooseVersion("1.12.1"):
+            self.target_opts += ["--config=nogcp"]
+            if LooseVersion(self.version) < LooseVersion("2.18"):
+                self.target_opts += ["--config=noaws", "--config=nohdfs"]  # works in 2.17.1 not in 2.18.1
```

Comment:
The comment is a bit odd given the checked versions. Better:
```suggestion
                self.target_opts += ['--config=noaws', '--config=nohdfs']  # Removed in 2.18
```

I found https://github.com/tensorflow/tensorflow/pull/72475 as a related PR upstream although this isn't the one where the support for the option was removed.